src/Client/Adapter/AdapterInterface.php

```diff
@@ -25,6 +26,7 @@ public function setOptions($options = []);
      * @param string  $host
      * @param int     $port
      * @param  bool $secure
+     * @return void
```

Comment:
Please revert, there is no native return type on this method, so you are deleting useful information